diff --git a/project/generic-arm-inc.mk b/project/generic-arm-inc.mk
index b4ef960..ad31077 100644
--- a/project/generic-arm-inc.mk
+++ b/project/generic-arm-inc.mk
@@ -103,7 +103,7 @@ KERNEL_CFI_ENABLED := false
 endif
 
 # Storage should send error reports to the metrics service
-STORAGE_ENABLE_ERROR_REPORTING := true
+STORAGE_ENABLE_ERROR_REPORTING := false
 STORAGE_AIDL_ENABLED ?= true
 
 ifeq ($(shell expr $(DEBUG) \>= 2), 1)
@@ -152,7 +152,6 @@ endif
 #
 MODULES += \
 	trusty/kernel/lib/trusty \
-	trusty/kernel/lib/metrics \
 	trusty/kernel/services/apploader \
 	trusty/kernel/services/hwrng \
 
@@ -211,7 +210,6 @@ TRUSTY_BUILTIN_USER_TASKS := \
 	trusty/user/app/storage \
 	trusty/user/base/app/apploader \
 	trusty/user/base/app/device_tree \
-	trusty/user/base/app/metrics \
 	trusty/user/base/app/system_state_server_static \
 
 MODULES += \
